### PR TITLE
Replaced Lerna with Turborepo

### PR DESCRIPTION
**Pull request by BurnedChris**

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * @type {import('prettier').Config}
+ */
+module.exports = {
+  ...require('@everfund/prettier/.prettierrc.json'),
+  // proseWrap: 'always', // Uncomment this while working on Markdown documents. MAKE SURE TO COMMENT THIS BEFORE RUNNING CHECKS/FORMATS OR EVERYTHING WILL BE MODIFIED.
```
Comment (lancscoder) [inline .prettierrc.js:6]: Can this be removed?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
-<p>
-    <a href="https://docs.everfund.io/" target="_blank">
-      <img alt="Everfund" width="350" src="https://ik.imagekit.io/everfund/everfund-sticker_U4p0fY8gw.svg">
-    </a>
-</p>
+# `Turborepo` Vite starter
```
Comment (lancscoder) [inline README.md:1]: I think we should be using the original Readme file

File: packages/js-sdk/src/index.ts [hunk 3]
```diff
@@ -233,61 +238,66 @@ class EverfundClient {
...
-    )
+    );
   }
 
   private setupIframeListeners() {
     window.addEventListener(
       'message',
       function (e) {
-        const embed = document.querySelector('.' + 'embedContainer')
+        console.log(e,e.data.message)
```
Comment (lancscoder) [inline packages/js-sdk/src/index.ts:264]: I think this should be removed

File: packages/js-sdk/src/index.ts [hunk 4]
```diff
@@ -181,47 +184,49 @@ class EverfundClient {
...
-      })
+      });
 
-      embedContainer.className = `embedContainer ${cssEmbedContainer}`
-      embedContainer.appendChild(loadingSpinner)
-      embedContainer.appendChild(modalWrap)
+      embedContainer.className = `embedContainer ${cssEmbedContainer}`;
+      embedContainer.appendChild(loadingSpinner);
+      embedContainer.appendChild(modalWrap);
       // disableBodyScroll(embedContainer)
```
Comment (lancscoder) [inline packages/js-sdk/src/index.ts:192]: This line should be removed

File: examples/next/package.json [hunk 5]
```diff
@@ -3,23 +3,24 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "examples:build": "vite build",
```
Comment (lancscoder) [inline examples/next/package.json:6]: should these not still be `next`?

File: .prettierrc.js (hunk 1 above)
Comment (BurnedChris) [reply .prettierrc.js:6]: This has to be kept so vscode plays nice!

**Pull request merged**